### PR TITLE
Desktop version portfolio

### PR DESCRIPTION
**Pull request by wolterbwambale**
This is my desktop portfolio version, I added the requirement HTML and CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,13 +189,14 @@ <h1 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h1>
                 <hr class="line">
             </li>
         </ul>
+     </div>
     </section>
     <!--contact-us-form-->
     <section class="contact-us">
```
Comment (souad988) [inline index.html:195]: Good job here as well your form is centered and section looks good on desktop , **but** Kindly check the background image in this section on figma design and make sure to add it and replicate the given template as much as possible

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @wolterbwambale , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔Good use of github workflow
✔Good PR title
✔No linter issues

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-🙏 Changing your PR summary to something better describing the changes done in this feature 😊

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wolterbwambale** [comment]
@souad988 I have add mobile background and worked on responsiveness and replacing the background as required.

File: styles.css [hunk 2]
```diff
@@ -365,18 +462,17 @@ nav {
...
   background: #6070ff;
   border-top-left-radius: 50px;
-  background-image: url(/img/contact-form-mobile-bg.png);
+  background-image: url(/img/contact-bg.png);
   background-repeat: no-repeat;
-  background-size: contain; 
+  background-size: contain;
   background-position: right;
-  
 }
```
Comment (Gambit142) [inline styles.css:476]: - [ ] Great work making some changes to the `background-image` of the Form section. However, I noticed that the `background-image` used is not the correct one and doesn't correlate with that of the Figma template used. Also, the positioning of the `background-image` is not uniform with that of the Figma template chosen. 👍 

### Your Design

Screenshot1

### Figma Template

Screenshot2

File: styles.css [hunk 3]
```diff
@@ -41,11 +61,12 @@ nav {
...
 
 .container {
-  background: url("/img/bg.jpeg");
+  background: url("/img/Header\ bg.png");
   background-size: 100% 100%;
   background-repeat: no-repeats;
```
Comment (Gambit142) [inline styles.css:66]: - [ ] Great work making some changes to the `background-image` of the _**top**_ section. However, I noticed that the `background-image` used is not the correct one and doesn't correlate with that of the Figma template used. Also, the positioning of the `background-image` is not uniform with that of the Figma template chosen. 👍 

### Your Design

Screenshot3

### Figma Template

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @wolterbwambale 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- All linters checks passed ✔️ 
- Use of media query between _**768px**_ to _**1440px**_ ✔️ 
- _**Hamburger**_ button doesn't show on the desktop version ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wolterbwambale** [comment]
@Gambit142  I have changed the background for mobile and contact me form.

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @wolterbwambale 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**